vaults/operator-deploy/vault-k8s-env-orchestrate-demo.yaml

```diff
@@ -105,7 +105,7 @@ spec:
           # Allow every pod in the default namespace to use the secret kv store
           - name: tx-signer
             bound_service_account_names: ["tx-signer", "vault-secrets-webhook", "vault"]
-            bound_service_account_namespaces: ["operator", "orchestrate-demo"]
+            bound_service_account_namespaces: ["operator", "vault-sandbox", "orchestrate-demo"]
```

Comment:
If we have replaced `orchestrate-demo` namesapce by ` <VaultKubernetesNameSpace>`, should we do the same here?